### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:

- Add some docs 

## Security considerations

None

### CODE REVIEW

File: docs/ops.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Operational Docs
+
+## Cleaning up a failed migration
+
+Use these instructions if a migration fails, and you want to retry it
+(e.g. you found and fixed the issue that caused it to fail).
+*note: if the migrator called `update` on the migration instance, this could get much more complicated, and fixing forward might be better*
```
Comment (ccostino) [inline docs/ops.md:7]: What do you mean by fixing forward?  Is this referring to steps 3 and later, or something else?

File: docs/ops.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+1. Get the guid of the migration service instance:
+   ```
+   $ cf service <instance-name> --guid
+   ```
+2. Purge the new service instance:
+   ```
+   $ cf purge-service-instance <migration instance name>
+   ```
+3. If the migrator called `update` on the migration instance, you'll need to
```
Comment (ccostino) [inline docs/ops.md:17]: Is step 3 just referring to moving along to steps 4 and 5, or is there a part missing?

**Review by ccostino — COMMENTED** [review]
🎉  Thanks, @bengerman13!  I had a couple of questions I wasn't sure about.

File: docs/ops.md (hunk 2 above)
Comment (bengerman13) [reply docs/ops.md:17]: step 3 is what happens when you switch tasks mid-sentence 👎

File: docs/ops.md (hunk 1 above)
Comment (bengerman13) [reply docs/ops.md:7]: fixing forward as in finish the migration by hand

**Review by apburnes — APPROVED** [review]

**Pull request merged**